pkgs/build-support/add-hardware-runpath/default.nix

```diff
@@ -0,0 +1,14 @@
+{ lib, stdenv }:
+
+stdenv.mkDerivation {
+  name = "add-hardware-runpath";
```

Comment:
Perhaps `add-driver-runpath` would be a more fitting name. We're not adding hardware into the runpath somehow but the driver thereof.

Reply:
that's fair